lib/create_facility.rb

```diff
@@ -0,0 +1,26 @@
+class CreateFacility
+
+    def initialize
+        @facility_info = []
+    end
+
+    def create_facilities(facility_info)
```

Comment:
Nice dynamic function! A switch statement may be considered here to improve readability of a messy ifelse.